github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-postsubmits.yaml

```diff
@@ -0,0 +1,59 @@
+postsubmits:
+  k8snetworkplumbingwg/kubemacpool:
+    - name: main-kubemacpool
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-kubevirtci-quay-credential: "true"
+      branches:
+        - main
+      cluster: phx-prow
+      spec:
+        nodeSelector:
+          type: vm
+          zone: ci
```

Comment:
I think this is no longer true @fgimenez for a job that is not starting kubevirtci instead it's building a container what kind of config we need ?

Reply:
ack, thanks!